### PR TITLE
Upload benchmark results for every test-models workflow (excl. Vulkan)

### PR DESCRIPTION
**Pull request by monorimet**

### CODE REVIEW

**Comment by monorimet** [comment]
try:
```
gsutil cp gs://iree-shared-files/nod-perf/bench_results/2022-08-04/ .
vim bench_results_gpu_da5b.csv
vim bench_results_cpu_da5b.csv
```

File: .github/workflows/test-models.yml [hunk 1]
```diff
@@ -75,17 +80,21 @@ jobs:
       if: matrix.suite == 'cpu'
       run: |
         cd $GITHUB_WORKSPACE
-        PYTHON=python${{ matrix.python-version }} ./setup_venv.sh
+        PYTHON=python${{ matrix.python-version }} IMPORTER=1 ./setup_venv.sh
         source shark.venv/bin/activate
-        pytest -k 'cpu' --ignore=shark/tests/test_shark_importer.py --ignore=benchmarks/tests/test_hf_benchmark.py --ignore=benchmarks/tests/test_benchmark.py 
+        pytest --benchmark -k 'cpu' --ignore=shark/tests/test_shark_importer.py --ignore=benchmarks/tests/test_hf_benchmark.py --ignore=benchmarks/tests/test_benchmark.py 
+        gsutil cp ./bench_results.csv gs://iree-shared-files/nod-perf/bench_results/${DATE}/bench_results_cpu_${SHORT_SHA}.csv
+        rm bench_results.csv
```
Comment (powderluv) [inline .github/workflows/test-models.yml:87]: hopefully rm is not required :D

File: shark/shark_benchmark_runner.py [hunk 2]
```diff
@@ -149,8 +149,8 @@ def benchmark_all_csv(
     ):
         self.setup_cl(inputs)
         field_names = [
-            "platform",
             "model",
+            "platform",
```
Comment (powderluv) [inline shark/shark_benchmark_runner.py:153]: some formatter requires the re-order ?

**Review by powderluv — APPROVED** [review]
minor nits

File: .github/workflows/test-models.yml (hunk 1 above)
Comment (monorimet) [reply .github/workflows/test-models.yml:87]: Sending in a fix for this, as it shouldn't be there.

File: shark/shark_benchmark_runner.py (hunk 2 above)
Comment (monorimet) [reply shark/shark_benchmark_runner.py:153]: Just changing the column order so that model comes first. This should do the trick

**Pull request merged**